### DISCUSSION TITLE
I got Clover working with Windows 10 on SSD an HDD with Boot Manager present on HDD

### DISCUSSION THREAD

**Discussion by alan-rpi** [CloverHackyColor/CloverBootloader#674] [Q&A]
Today I got my PC working with Windows 10 on both a HDD and as SSD (using a non-bootable NVMe-PCIe adapter). 

It's taken some time as I needed to convert the HDD from MBR to GPT and in the processes ended up with Windows Boot Manager present on the HDD EFI partition.

I wrote up my issues and solution at [AskWoody](https://www.askwoody.com/forums/topic/rebuilding-the-windows-boot-manager-to-get-clover-to-run/#post-2629666)   Go straight down to my post on January 30, 2024 at 12:53 pm . This starts off with the conversion to GPT and formatting the SSD and HDD and ends with part installing Clover. The following post then completes the Clover installation.

What I've yet to get working is for Clover not to display all the non-bootable partitions in its menu screen - over 10 of them! I edited the config.plist file and added the partition IDs (labels) to the Hide key but it didn't work. Any ideas welcome.

`		<key>#Hide</key>`
`		<array>`
` 	                 <string>Win 10 SSD</string>                              <<< example entry added of SSD C: partition`
`			   <string>Win 10 HDD</string>                             <<<< ditto of HDD C: partition`
`		         <string>Windows</string>                                 <<<< config.plist example`
`	                  <string>BOOTX64.EFI</string>                            <<<< config.plist example`
`		</array>`

Win 10 SSD and Win 10 HDD are the C: partitions, not the associated EFI partitions. EFI partitions is what Clover uses. 

Alan

**Comment by eugenesvk** [comment]
`<key>#Hide</key>` means the key is commented out, i.e., inactive/ignored

`<key>Hide</key>` remove `#` to make it work

**Comment by alan-rpi** [comment]
Oh! what a simple oversight! Many thanks Eugene. 
PS Otherwise Clover has been running fine :)
